core_v2/sources/domains.move

```diff
@@ -1047,6 +1103,8 @@ module aptos_names_v2::domains {
             domain_name,
             new_expiration_time_sec - now,
             0,
+            option::none(),
+            option::none(),
         );
         // TODO: `register_name_internal` should accept a `target_addr`
```

Comment:
maybe remove this?

Reply:
we need to set them to satisfy the function signature

Reply:
I mean the comment